INSTALL.md

```
@@ -25,18 +25,24 @@ These are tested plugins and are great additions to the theme:
 
 ## REST API Endpoints
 
-The theme provides the following REST API endpoints:
+The theme provides the following REST API endpoint:
 
-### Tekst TV
+### Broadcast Data
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)